server/routes/api/babysitter/babysitterRoutes.ts

```diff
@@ -38,4 +52,129 @@ babysitterRouter.get(
   }
 );
 
-export default babysitterRouter;
+babysitterRouter.put(
+  "/image/:babysitterId",
+  [
+    param("babysitterId").notEmpty().isNumeric().withMessage(BABYSITTER_INVALID_INPUT_ERROR)
+  ],
+  upload.single('image'),
+  async (req: Request, res: Response) => {
+    try {
+      const fieldValidationResult = validationResult(req);
+      if (!fieldValidationResult.isEmpty()) {
+        return res.status(400).json({
+          message: fieldValidationResult
+            .array()
+            .map((item) => item.msg)
+            .join(" "),
+        });
+      }
+      const { babysitterId } = req.params;
+      const validation = await handler.userValidation(Number(babysitterId));
+      if (!validation.isValid) {
+        return res.status(400).json({ error: validation.message });
+      }
+
+      const imageName = `babysitter_${babysitterId}`;
+
+      if (!req.file) {
+        return res.status(400).json({ error: `Missing file input.` });
+      }
+      
+      const response = await s3.putImage(req.file, imageName);
+      if (!response) {
+        return res.status(500).json({ message: `Couldn't upload image` });
+      }
+      await handler.putProfileImage(imageName, Number(babysitterId));
+
+      return res.status(200).json({ message: `Image profile uploaded.` });
+    } catch (e) {
+      console.log(
+        `Error message: ${req.body.id}: ${(e as Error).message}\n${
+          (e as Error).stack
+        }`
+      );
+      return res.status(500).end();
+    }
+  }
+);
+
+babysitterRouter.get(
+  "/image/:babysitterId",
+  [
+    param("babysitterId").notEmpty().isNumeric().withMessage(BABYSITTER_INVALID_INPUT_ERROR)
+  ],
+  async (req: Request, res: Response) => {
+    try {
+      const fieldValidationResult = validationResult(req);
+      if (!fieldValidationResult.isEmpty()) {
+        return res.status(400).json({
+          message: fieldValidationResult
+            .array()
+            .map((item) => item.msg)
+            .join(" "),
+        });
+      }
+      const { babysitterId } = req.params;
+      const validation = await handler.userValidation(Number(babysitterId));
+      if (!validation.isValid) {
+        return res.status(400).json({ error: validation.message });
+      }
+
+      const imageName = await handler.getProfileImage(Number(babysitterId));
+      if (!imageName) {
+        return res.status(204).json({ message: `User doen't have an image` });
+      }
+      
+      const localStackImageUrl = await s3.getImageUrl(imageName);
```

Comment:
then here i would create an handler function that gets the imageName and gets the value for the s3.getImageUrl here and returns it's value